experiment_buddy/experiment_buddy.py

```diff
@@ -234,17 +238,39 @@ def _setup_tb(logdir):
     return tensorboardX.SummaryWriter(logdir=logdir)
 
 
-def _ensure_scripts(extra_headers):
+def _ssh_session_connected():
+    """ TODO: move this to utils.py or to a class (better) """
+
+    kwargs_connection = {}
+
+    try:
+        kwargs_connection["host"] = os.environ["hostname"]
+    except KeyError:
+        raise EnvironmentError("Please add your hostname as env:" "\nhostname")
+
     try:
-        host = os.environ["MY_BUDDY_HOSTNAME"]
-        passwd = os.environ["MY_BUDDY_PASSWORD"]
+        kwargs_connection["connect_kwargs"] = {"password": os.environ["password"]}
```

Comment:
make it BUDDY_PASSWORD so we don't risk to overwrite other "password" env variables.